packages/website/components/account/filesManager/uploadsTable.js

```diff
@@ -40,43 +40,54 @@ const defaultQueryOrder = `${defaultSortBy},${defaultSortOrder}`;
  * @param {UploadsTableProps} props
  */
 const UploadsTable = ({ content, hidden, onFileUpload, onUpdatingChange, showCheckOverlay }) => {
-  const { uploads, pages, fetchDate, getUploads, isFetchingUploads, deleteUpload, renameUpload } = useUploads();
-  const {
-    query: { filter },
-    query,
-    replace,
-  } = useRouter();
+  const { uploads, pages, count, fetchDate, getUploads, isFetchingUploads, deleteUpload } = useUploads();
+
+  const totalUploads = count;
+  const { query, replace } = useRouter();
   const {
     storageData: { refetch },
     info,
   } = useUser();
 
-  const [sortBy, setSortBy] = useState(defaultSortBy);
-  const [sortOrder, setSortOrder] = useState(defaultSortOrder);
-  const [page, setPage] = useState(1);
-  const [size, setSize] = useState(10);
-  const [linkPrefix, setLinkPrefix] = useState('w3s.link/ipfs/');
-  const [keyword] = useState(filter);
+  // Strings
+  const fileRowLabels = content?.table.file_row_labels;
+  const statusMessages = fileRowLabels.status.tooltip;
+
+  // Constants
+  const defaultLinkPrefix = 'w3s.link/ipfs/';
+  const defaultSortBy = 'Date';
+  const defaultSortOrder = 'Desc';
+  const defaultQueryOrder = `${defaultSortBy},${defaultSortOrder}`;
+  const defaultSize = ROWS_PER_PAGE_OPTIONS[0];
+
+  // Query calculated defaults
+  const defaultPage = Array.isArray(query.page) || query.page === undefined ? 1 : parseInt(query.page);
```

Comment:
Not sure I understand why we have the first check. 
If that is an array, wouldn't assigning a booleanto `defaultPage`🤔

Reply:
Fixed this one.

Reply:
That's a good point! I think you have fixed this by defaulting to the first value, thanks!